modules/claims_api/app/controllers/claims_api/v2/blueprints/claim_blueprint.rb

```diff
@@ -47,6 +47,23 @@ class ClaimBlueprint < Blueprinter::Base
         end
         field '5103_waiver_submitted'.to_sym
 
+        field :tracked_items do |claim, options|
+          bgs_service = BGS::Services.new(external_uid: options[:ptpct_id],
+                                          external_key: options[:ptpct_id])
+          bgs_response = bgs_service.tracked_items.find_tracked_items(claim[:claim_id])
+          bgs = bgs_response.dig(:benefit_claim, :dvlpmt_items) || []
+          bgs.map do |item|
+            {
+              closedDate: item[:jrn_dt].iso8601,
+              description: item[:short_nm],
+              dvlpmtTc: item[:dvlpmt_tc],
+              suspensionDate: item[:suspns_dt].iso8601,
+              requestedDate: item[:req_dt].iso8601,
+              trackedItemId: item[:dvlpmt_item_id].to_i
+            }
+          end
+        end
```

Comment:
We probably don't want to put this logic in the blueprint.
Rather, we should probably move it over into [here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb#L134).